### PR TITLE
QA test environment for nav

### PR DESCRIPTION
**Pull request by Heleen-cr**
**Test notes:**

Default navigation:
- Has all items inline in the header from the desktop-nav breakpoint on (1150px).
- The hamburger menu button is shown instead on smaller screen sizes.
- On desktop-nav breakpoint sub-items are shown on hover on the menu-items.
- On smaller screen sizes the sub-items appear on click on the menu-item.
- If a menu-item doesn't have sub-items the menu-item itself should follow the link on click.
- Check if the meta icons still appear properly and the interaction with them still works.

Feature navigation can be enabled by enabling the cr_navigation module

### CODE REVIEW

File: themes/custom/campaign_base/sass/styles.scss [hunk 1]
```diff
@@ -7,3 +7,5 @@ $image-path: "/profiles/contrib/cr/themes/custom/campaign_base/images/patternlab
 @import "@comicrelief/pattern-lab/sass/base/_variables";
 @import "@comicrelief/pattern-lab/sass/base/_core";
 @import "@comicrelief/pattern-lab/sass/base/_components";
+
+//test change do not merge me :)
```
Comment (pvhee) [inline themes/custom/campaign_base/sass/styles.scss]: @Heleen-cr @AndyEPhipps note that you can also use the develop environment on platform.sh for this, which should contain the latest `develop`. This is equal to what you're doing here but without adding a new env

Comment (Heleen-cr) [inline themes/custom/campaign_base/sass/styles.scss]: yes well the thing is that the latest pattern-lab changes didn't show on the develop branch (even though plat.sh rebuilt)

**Comment by Heleen-cr** [comment]
@k-pammi @pradip3271 
Note: NO merging please :)

File: themes/custom/campaign_base/sass/styles.scss (hunk 1 above)
Comment (pvhee) [inline themes/custom/campaign_base/sass/styles.scss]: @Heleen-cr interesting! let's research.. maybe `develop` wasn't automatically updated after the merge? @Saphyel @adamclark-dev could any of you have a quick look?

Comment (Saphyel) [inline themes/custom/campaign_base/sass/styles.scss]: Looking into it

**Comment by k-pammi** [comment]
@Heleen-cr Raised bugs in https://github.com/comicrelief/campaign/issues/1057

**Pull request closed without merging**